### DISCUSSION TITLE
URGENT PLS HELP

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#107] [Q&A]
I am 1000% sure i am typing the correct password. It is stuck at logon screen after entering password

**Comment by DiegoSof2** [comment]
did you check your keyboard layout?

**Reply by sudobrat** [reply]
Yes it is allowing log in in terminal but not on desktop

**Reply by DiegoSof2** [reply]
Something similar happened to me, in this case I tried to install several times and it worked, it seemed that I was installing in parts and I had to issue the install command several times

**Reply by sudobrat** [reply]
Is there a way to install without nvidia ? I have nvidia but i have heard nvidia does not go well with wayland and i think that is causing a lot of bt

**Reply by DiegoSof2** [reply]
I really don't know, I'm a beginner in linux, the problem I had for using Nvidia was after booting the installation the screen was black, so I had to configure in glub to open in a kind of safe mode, you can try to connect the HDMI card onboard and disable external card through bios

**Comment by prasanthrangan** [comment]
Hi @GeekWolf007, can you please try login from TTY, and you should be able to see the error.

**Reply by T-Crypt** [reply]
@GeekWolf007 try booting with nomodeset set. If this isn't about the ISO but still about nvidia with the prop driver try nvidia-drm.modeset=1 instead.

should be able to get to TTY by editing grub on boot even with nvidia prop installed

**Reply by T-Crypt** [reply]
@GeekWolf007 

Make sure on the login screen for the first login that you have input your username / user account and then enter your password. Had a similar error where I put in password and it just looped until I entered my username.
